### DISCUSSION TITLE
Dra og slipp bokser til en søppelbøtte for å slå dem av

### DISCUSSION THREAD

**Discussion by forteller** [entur/tavla#320] [Ideas]
Når man lager en ny tavle kan den ofte starte med veldig mange bokser. Det er mye lettere å få oversikt over hvilke av dem man ønsker å ha og ikke på selve tavla med boksene enn i innstillingene, så det hadde vært fint  bare med én gang kunne drag-n-drop direkte til en søppelbøtte for å fjerne/slå av de man ikke ønsker, istedenfor å måtte gå inn i innstillingene og finne igjen riktige bokser bare utfra navnene og slå dem av der.

**Comment by lindtvedtsebastian** [comment]
This discussion is linked to the old (deprecated) version of Tavla and will therefore be closed.
